### ISSUE TITLE
测试

### ISSUE THREAD

**Issue by zuqj** [zuqj/test1#2]
```javascript
function fancyAlert(arg) {
  if(arg) {
    $.facebox({div:'#foo'})
  }
}
```

First Header | Second Header
------------ | -------------
Content from cell 1 | Content from cell 2
Content in the first column | Content in the second column

16c999e8c71134401a78d4d46435517b2271d6ac
mojombo@16c999e8c71134401a78d4d46435517b2271d6ac
mojombo/github-flavored-markdown@16c999e8c71134401a78d4d46435517b2271d6ac

**Comment by zuqj** [comment]
fix

**Comment by zuqj** [comment]
fix #2

**Comment by zuqj** [comment]
close #2
